src/WebJobs.Script/Workers/IFunctionInvocationDispatcher.cs

```diff
@@ -25,8 +25,6 @@ public interface IFunctionInvocationDispatcher : IDisposable
 
         Task<bool> RestartWorkerWithInvocationIdAsync(string invocationId);
 
-        Task FinishInitialization(IEnumerable<FunctionMetadata> functions, CancellationToken cancellationToken = default);
-
         Task<IEnumerable<RawFunctionMetadata>> GetWorkerMetadata();
```

Comment:
Remove `GetWorkerMetadata()` as it is not being used.

Reply:
done